src/vs/workbench/parts/files/electron-browser/views/explorerViewer.ts

```diff
@@ -316,6 +320,27 @@ export class FileRenderer implements IRenderer {
 	}
 }
 
```

Comment:
I feel like this should not be a function, either a lambda expression or a private method in the explorerViewer. Though if you prefer functions we can leave it this way